app/__init__.py

```diff
@@ -28,18 +30,19 @@ def null_to_dash(value):
     return value
 
 
+# use only for local development
+if os.environ.get("DEFAULT_AWS_PROFILE"):
```

Comment:
moved here so that we ensure that we have set the boto3 session to the right profile by the time we access any of the config properties which depend on aws.